kip-0019.md

```diff
@@ -0,0 +1,75 @@
+---
+KIP: 0019
+Title: Coin-v6 and fungible-v3
+Author: Robert Soeldner (robert@kadena.io)
+Status: Draft
+Type: Standard
+Category:
+Created: 2023-05-02
+---
+
+## Abstract
+
+The next iteration of the coin contract proposes three main changes to enhance
+the _maintainability_ and _security_ of the contract.
+
+Firstly, the `rotate` function, which is responsible rotating an account guard,
+will be removed to preserve cross-chain soundness. This is to avoid the risk locking funds.
```

Comment:
Should the proposal give an explicit time table for deprecating `rotate` and forcing users to adopt a workaround?